app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java

```diff
@@ -69,6 +70,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         } catch (ParseException e) {
             Timber.e("Parsing Error Occurred at BookmarksFragment::onCreateView.");
         }
+        if(!bookmarkedIds.isEmpty())
+        {
+            noBookmarks.setVisibility(View.GONE);
```

Comment:
I was thinking of doing this by a bus event but didn't do it for the time being. Right now the view is refreshed everytime it is created.